### ISSUE TITLE
Provide a default function compatible with ES6 module import

### ISSUE THREAD

**Issue by yanickrochon** [caub/base-conv#2]
Currently, the provided example suggest :

```js
const conv = require('base-conv')();
```

But this form is not compatible with `import` statements. So one must do :

```js
import BaseConv from 'base-conv';
const conv = BaseConv();
```

There should be an export of `conv` directly. In the source code :

```
module.exports.conv = BaseConv();
```

So the module can be imported as :

```js
import { conv } from 'base-conv';
```

**Comment by caub** [comment]
We could export a default instance, but what would be the input and output alphabet? '0123456789abcdef' / '0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ-_' ?

right, let's do so, thanks for the suggestion

**Comment by yanickrochon** [comment]
The default ones. If users want another default, then can simply import `BaseConv` instead and provide their own. My point is, for most cases, the defaults are fine, so not offering a default option adds an extra step for most use cases, and especially so when using `import`.

**Comment by caub** [comment]
Ok, thanks, fixed in 66faa09593d66cc706baa47d38d7458f87374a5d
